### PR TITLE
[Bug] ContentPage.ControlTemplate gets not rendered in Wpf (#7671)

### LINKED ISSUE #7671: [Bug] ContentPage.ControlTemplate gets not rendered in Wpf
### Description

Only the `Content` property is rendered in the Wpf PageRenderer.

The solve this, it would be the best to render the first child. PR will follow.

### Steps to Reproduce

Add a control template to a `ContentPage`.

### Expected Behavior

The control template gets rendered.

### Actual Behavior

Only the `Content` gets rendered.

### Basic Information

This issue is only in Wpf present.

### PR DESCRIPTION
**Pull request by neolithos**
### Description of Change ###
`PageRenderer` renderes the first child instead of the `Content`.

Sorry, I destroyed the branch #7672.

### Issues Resolved ### 
- fixes #7671

### API Changes ###
 None

### Platforms Affected ### 
- WPF

### Behavioral/Visual Changes ###
ControlTemplate gets rendered in wpf.

### Before/After Screenshots ### 

Before:
PageRendererBefore
After:
PageRendererAfter

### PR Checklist ###
- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by neolithos** [comment]
Is there a better solution?

Because in the `FormsContentLoader` it looks like, that someone had take care of ControlTemplates and Pages some how? But I had to change this a line for the layout.

Please, have a closer look to my changes.

File: Xamarin.Forms.Platform.WPF/Controls/FormsContentPage.cs [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+using System.Windows;
+using WpfSize = System.Windows.Size;
+
+namespace Xamarin.Forms.Platform.WPF.Controls
 {
 	public class FormsContentPage : FormsPage
 	{
 		public FormsContentPage()
 		{
-			this.DefaultStyleKey = typeof(FormsContentPage);
```
Comment (neolithos) [inline Xamarin.Forms.Platform.WPF/Controls/FormsContentPage.cs]: Why not using the FrameworkPropertyMetadata?

**Comment by neolithos** [comment]
What is missing? What is wrong?

**Comment by jsuarezruiz** [comment]
@neolithos Could you rebase master to fix the conflict, please?

**Comment by neolithos** [comment]
That looks strange... Is this okay?

**Comment by neolithos** [comment]
Do I have something to do?

**Comment by VladislavAntonyuk** [comment]
@neolithos please resolve conflicts

**Comment by neolithos** [comment]
I rebased this a more than ones since 2019.

But how do I solve this:
```
! [remote rejected]     issue-7671 -> issue-7671 (refusing to allow an OAuth App to create or update workflow `.github/workflows/dotnet-format-daily.yml` without `workflow` scope)
```

**Comment by jfversluis** [comment]
Epic! It took a little while (🤏 ) but finally got to look at this. It works for the Xamarin Community Toolkit sample app! Thanks for this great contribution!

**Comment by neolithos** [comment]
That was long...

**Comment by jfversluis** [comment]
At least it wasn't never! 🎉

**Pull request merged**